subsys/bluetooth/mesh/net.h

```diff
@@ -268,6 +268,7 @@ struct bt_mesh_net_rx {
 	       net_if:2,       /* Network interface */
 	       local_match:1,  /* Matched a local element */
 	       friend_match:1; /* Matched an LPN we're friends for */
+	u16_t  msg_cache_idx;  /* Index of entry in message cache */
```

Comment:
Unrelated to this PR, but potential future improvement work: this struct used to have more members originally and the `u8_t` bitfield made sense then. Now there's unnecessary padding, and we could actually move the fields into the preceding `u32_t` since the sequence number is a 24 bit value in practice. I'd then also just use `int` for the cache index since the struct doesn't become any bigger due to it and it will generate slightly more efficient code for 32 bit MCUs. Same goes for the functions iterating the cache, i.e. change `u16_t` to `int` in `net.c`. Additional bonus with this is getting rid of the `U` suffix from the likes of `0U` (something I rather dislike, but which is a MISRA-C requirement for assignments to unsigned integer variables).